cogs/public.py

```diff
@@ -78,7 +78,31 @@ async def serverinfo(self, ctx: commands.Context):
         """
         Present server information.
         """
-        raise NotImplementedError('Command requires implementation and permission set-up.')
+        embed = discord.Embed(title=f"{ctx.guild.name} ({ctx.guild.id})", color=discord.Color.green())
+        creation_date = ctx.guild.created_at
+        # unix timestamp
+        timestamp = int(creation_date.timestamp())
+        embed.add_field(name="**Creation Date**", value=f"<t:{timestamp}> (<t:{timestamp}:R>)", inline=False)
```

Comment:
Probably good to use `discord.utils.format_dt` here (and if you are down for the task, replace existing uses of the project's `utils.time.format_timestamp` with `discord.utils.format_dt` and delete the former).